digiwf-libs/digiwf-spring-logging-and-tracing/pom.xml

```diff
@@ -9,8 +9,7 @@
         <version>1.3.0-SNAPSHOT</version>
     </parent>
 
-    <artifactId>digiwf-logging</artifactId>
-    <version>1.3.0-SNAPSHOT</version>
+    <artifactId>digiwf-spring-logging-and-tracing</artifactId>

```

Comment:
`digiwf-logging` wird schon in anderen Services verwendet. Wenn wir das hier umbenennen, müssen wir das auch in digiwf-cocreation und ggf. der Formserver Integration nachziehen

Reply:
werden wir dann müssen. ich hab den namen initial sehr unglücklich gewählt